backend/.env.example

```
@@ -30,3 +30,14 @@ SECRET_KEY=Secret123OwnKey25012021
 SENDGRID_API_KEY = yourApiKey
 SENDGRID_EMAIL_FROM = email
 
+#
+# NODEMAILER
+#
+NODEMAILER_MAIL=db.gravy.2021@gmail.com
+NODEMAIL_PASS=dbgravy2021
```

if it's a real password, please remove it from here
```suggestion
NODEMAIL_PASS=
```